tsconfig.json

```diff
@@ -26,16 +26,7 @@
     "strict": true,
     "paths": {
       "@hive/api": ["./packages/services/api/src/index.ts"],
-      "@hive/api/src/modules/schema/providers/artifact-storage-reader": [
-        "./packages/services/api/src/modules/schema/providers/artifact-storage-reader.ts"
-      ],
-      "@hive/api/src/modules/shared/providers/redis": [
-        "./packages/services/api/src/modules/shared/providers/redis.ts"
-      ],
-      "@hive/api/src/modules/shared/providers/pub-sub": [
-        "./packages/services/api/src/modules/shared/providers/pub-sub.ts"
-      ],
-
+      "@hive/api/*": ["./packages/services/api/src/*"],
```

Comment:
Allowing wildcard imports seemed to be more flexible until we figure out how we want to organize shared dependencies (e.g. break them into their own packages).

Reply:
I am not super strongly opinionated if this makes this easier for you, but I tend to prefer keeping a single entry point because I think the restrictions force you to make better decisions around package boundaries